.devcontainer/cuda11.8-conda/devcontainer.json

```diff
@@ -5,9 +5,14 @@
     "args": {
       "CUDA": "11.8",
       "PYTHON_PACKAGE_MANAGER": "conda",
-      "BASE": "rapidsai/devcontainers:24.06-cpp-llvm16-cuda11.8-mambaforge-ubuntu22.04"
+      "BASE": "rapidsai/devcontainers:24.06-cpp-cuda11.8-mambaforge-ubuntu22.04"
     }
   },
+  "runArgs": [
+    "--rm",
+    "--name",
+    "rapids-${localWorkspaceFolderBasename}-24.06-cuda11.8-conda"
```

Comment:
I have wondered if we should somehow put the external username in the name so that you can tell when running on a server (DGX) which containers are yours. I often have trouble figuring out which ones are mine. Is there some other way?

Reply:
That's a good point, I forgot about the shared-DGX use-case. I'll see if I can inject the username (or at least `$RANDOM`) into the container name.

The container's hostname should match the `CONTAINER ID` column in the `docker ps` output.

In the container:
```shell
(rapids) coder ➜ ~ $ hostname -A
7930522bd3d6 
```

On the host:
```shell
$ docker ps -a
CONTAINER ID   IMAGE                                                                           COMMAND                  CREATED             STATUS             PORTS     NAMES
7930522bd3d6   vsc-cudf-0f5585425caf1a42951009736220a75958901c70e7d17a0882db1ab6499d0605-uid   "/bin/sh -c 'echo Co…"   8 seconds ago       Up 7 seconds                 rapids-cudf-24.06-cuda12.2-conda
```

Reply:
Seems like adding `${localEnv:USER}-` to the container name does work. I'll update these PRs.

Reply:
Confirmed this works on a DGX over Remote SSH:
```shell
$ docker ps -a
CONTAINER ID   IMAGE                                                                                    COMMAND                  CREATED         STATUS                  PORTS                                       NAMES
db9b3e068cbf   vsc-rmm-536ac2b43c9291d1f5ce41e7d65c1793adfacaa23acf4dacd3f314fba894aae3-uid             "/bin/sh -c 'echo Co…"   2 minutes ago   Up 2 minutes                                                        pataylor-rapids-rmm-24.06-cuda11.8-conda
```